CLOUD-2699 - Remove EAP 7.0 templates, update EAP templates to latest release

luck3y opened the pull request:
See: https://issues.jboss.org/browse/CLOUD-2699
Signed-off-by: Ken Wills kwills@redhat.com

bparees commented on official/eap/imagestreams/jboss-eap70-openshift-rhel7.json:
```diff
@@ -1,109 +0,0 @@
-{
```
this change appears to be dropping the eap70 imagestreams, which isn't allowed

luck3y replied:
Yep, it is dropping them as I didn't know it wasn't allowed ;) I'll update it to keep those.

bparees replied:
yeah so we never remove an imagestream because if we remove an imagestream from openshift then resources that reference that imagestream would immediately become broken.

luck3y replied:
@bparees Yeah, it seems obvious now you've pointed it out :) Thanks.

luck3y commented:
@bparees Updated to restore the eap70 imagestream, and we'll keep it around upstream as well.

luck3y commented:
Also, WRT to the CI failure, running make import updates these too:

modified:   official/fis/imagestreams/fis-java-openshift-rhel7.json
modified:   official/fis/imagestreams/fis-karaf-openshift-rhel7.json

But I didn't feel comfortable including these in this change. Is there someone I should contact regarding those?

bparees commented:
> But I didn't feel comfortable including these in this change. Is there someone I should contact regarding those?

this repo auto-imports every night, so if you don't include them, they're going to get picked up anyway.  (it's also possible you're picking them up because you're not rebased on the latest master?)

Anyway you should include them to make our CI happy.

luck3y commented:
@bparees Done, thanks Ben. I kept those as a separate commit, so feel free to squash or whatever :)
FWIW, I was already rebased with master, so it must just be they hadn't been imported yet or something.

bparees commented on official/eap/imagestreams/jboss-eap71-openshift-rhel7.json:
```diff
@@ -2,71 +2,89 @@
+                    "name": "registry.access.redhat.com/jboss-eap-7/eap71-openshift:1.1"
                 },
-                "name": "1.0-TP"
+                "name": "1.1"
```
hm, this is still dropping an imagestreamtag.  but it was a tech preview one so maybe that's ok.. @luciddreamz any opinion?

luck3y replied:
@bparees Yeah, we had a ticket on that one asking us to remove it as it would cause import issues, I'll find the ticket.

luck3y replied:
@bparees https://github.com/jboss-openshift/application-templates/pull/450#issuecomment-393299676

https://issues.jboss.org/browse/CLOUD-2594

But if this will cause issues, let me know and we can revisit.

luck3y replied:
Ah, yeah that's right the images themselves don't exist anymore.

bparees replied:
that seems like a good enough reason to remove it :)

bparees commented:
lgtm.  I've posted a comment on https://github.com/jboss-fuse/application-templates/commit/400bea1de0fd576e5ab4f256f2c314268a1c0a0c#diff-1346857a7ca6f54f340c9dbe677d99e3 about the FIS imagestreamtag removal that needs to be reverted.
